src/Meth.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+const Meth = () => {
+  // State hook to manage an array of numbers
+  const [numbers, setNumbers] = useState([1, 2, 3, 4, 5]);
+
+  // Adding a new number to the array using the push method
+  const addNumber = () => {
+    const newNumber = Math.floor(Math.random() * 10) + 1;
+    numbers.push(newNumber);
+    setNumbers([...numbers]); // Triggering a state update to reflect the change
+  };
+
```

Comment:
Hi @Showcode24. This is an awesome implementation of JavaScript methods. Kindly ensure that beyond researching and getting something to submit, you learn each of the methods also by heart so it can be handy when needed.

**Good work!**